src/for_2D_build/bodies/unstructured_mesh.cpp

```diff
@@ -1,19 +1,68 @@
 
-#include "common_shared_FVM_classes.h"
+#include "unstructured_mesh.h"
+
 namespace SPH
 {
 //=================================================================================================//
-void readMeshFile::getDataFromMeshFile()
+ANSYSMesh::ANSYSMesh(const std::string &full_path)
+{
+    getDataFromMeshFile(full_path);
+    getElementCenterCoordinates();
+    gerMinimumDistanceBetweenNodes();
+}
+//=================================================================================================//
+void ANSYSMesh::readNodeCoordinate(const std::string &text_line, StdLargeVec<Vec2d> &node_coordinates)
+{
+    size_t divide_position = text_line.find_first_of(" ");
+    std::string x_part = text_line;
+    std::string y_part = text_line;
+    std::string x_coordinate_string = x_part.erase(divide_position);
+    std::string y_coordinate_string = y_part.erase(0, divide_position);
+
+    Vec2d coordinate = Vec2d::Zero();
+    std::istringstream stream_x, stream_y;
+    stream_x.str(x_coordinate_string);
+    stream_y.str(y_coordinate_string);
+    stream_x >> coordinate[0];
+    stream_y >> coordinate[1];
+    node_coordinates.push_back(coordinate);
+}
```

Comment:
how about more compact way of 

size_t divide_position = text_line.find_first_of(" ");
Vec2d coordinate = Vec2d::Zero();
std::istringstream( text_line.erase(divide_position) ) >> coordinate[0];
std::istringstream( text_line.erase(0, divide_position) ) >> coordinate[1];

Reply:
@ZhentongWang Could you consider Chi comment for a later revision of the code?